docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1568 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    _ = gc.collect()
</pre><p>If you're trying to avoid showing the result, just write <code>gc.collect();</code> with a semicolon to prevent displaying the output. Assigning the result looks odd.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>